charts/opentelemetry-collector/examples/host-entity/rendered/configmap-agent.yaml

```diff
@@ -35,7 +35,7 @@ data:
             cx.agent.type: agent
             cx.cluster.name: test
             cx.integrationID: test
-            k8s.node.name: ${env:KUBE_NODE_NAME}
```

Comment:
There's no need to worrya bout this change here, because the env var with the node name is only set when the k8s resources attributes is enabled and this is not the case in this example.